ramses_inspector.py

```diff
@@ -22,28 +23,30 @@ def __init__(self,
                  scene: ExportableScene):
 
         self.scene = scene
-        self._window = None
+        self.viewer_process = None
 
-    def load_viewer(self):
+    def load_viewer(self, platform, block: bool = False):
         """Loads the RAMSES scene viewer for visual inspection of the
         generated scene"""
 
+        assert isinstance(platform, str)
+        platform = platform.lower()
+
         self.close_viewer()
 
         resolution_x = self.scene.blender_scene.render.resolution_x
         resolution_y = self.scene.blender_scene.render.resolution_y
 
-        self._window = self.scene.ramses.openWindow(resolution_x, resolution_y, 0, 0)
-        self._window.showScene(self.scene.ramses_scene)
+        program_args = f'-s {self.scene.output_path}{self.scene.blender_scene.name} -x screenshot.png -xw {resolution_x} -xh {resolution_y}'
```

Comment:
Use path.join() please... This breaks unexpectedly if the path is not with a trailing slash

Reply:
Also - I guess you want to open a viewer for preview here, or to take a screenshot? If the former - then you don't need to pass in the screenshot command line. If the latter - then you should probably specify the path of the screenshot in the method as a param (so that we can use the same in the tests)

Reply:
I was shooting for the former, but should we also take a screenshot by default?

Reply:
Then I'd say remove the screenshot params. I modified the scene viewer to terminate itself after it takes a screenshot - so that you don't have to kill the process yourself in the tests. I'd say you don't have to take a screenshot here

Reply:
Is 09ab350 any better? Similar to the strategy adopted in RAMSES I guess: take a screenshot if the path is non-null?